src/server/nuclearnet/fake_nuclearnet_server.ts

```diff
@@ -3,6 +3,7 @@ import { NUClearNetSend } from 'nuclearnet.js'
 import * as XXH from 'xxhashjs'
 
 import { createSingletonFactory } from '../../shared/base/create_singleton_factory'
+import { hashType } from '../../shared/nuclearnet/nuclearnet_proxy_parser'
```

Comment:
I don't know where this function should go, both of these places seem like a bad spot.
How do we not have a utility location yet :P

Reply:
Moved back :)

Reply:
> How do we not have a utility location yet :P

NEVER!!! 🙂